lms/static/js/groups/views/certificate_whitelist_editor.js

```diff
@@ -0,0 +1,86 @@
+// Backbone Application View: CertificateWhitelist View
+/*global define, RequireJS */
+
+;(function(define){
+    'use strict';
+
+    define([
+            'jquery',
+            'underscore',
+            'gettext',
+            'backbone',
+            'js/groups/models/certificate_exception'
+        ],
+
+        function($, _, gettext, Backbone, CertificateException){
+            return Backbone.View.extend({
+                el: "#certificate-white-list-editor",
+                error_div: '.error-message',
+
+                events: {
+                    'click #add-exception': 'addToWhiteList'
+                },
+
+                initialize: function(options){
+                    this.certificateWhiteList = options.certificateWhiteList;
+                },
+
+                render: function(){
+                    var template = this.loadTemplate('certificate-white-list-editor');
+
+                    this.$el.html(template());
+
+                },
+
+                loadTemplate: function(name) {
+                    var templateSelector = "#" + name + "-tpl",
+                    templateText = $(templateSelector).text();
+                    return _.template(templateText);
+                },
+
+                addToWhiteList: function(){
+                    var value = $("#certificate-exception").val();
```

Comment:
please always use more concise selectors `$(this).find("#certificate-exception")`.